contracts/whitelist/src/contract.rs

```diff
@@ -22,6 +23,7 @@ const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 const MAX_MEMBERS: u32 = 5000;
 const CREATION_FEE: u128 = 100_000_000;
 const MIN_MINT_PRICE: u128 = 25_000_000;
+const MAX_PER_ADDRESS_LIMIT: u64 = 30;
```

Comment:
If max members is a `u32`, then this type doesn't have to be larger than it right?

Reply:
you're right, changed to `u32`